.github/workflows/build-nightly.yml

```
@@ -44,7 +44,7 @@ jobs:
       - name: Generate version suffix
         id: version_vars
         run: |
-          echo mastodon_version_suffix=+nightly-$(date +'%Y%m%d') >> $GITHUB_OUTPUT
+          echo mastodon_version_suffix=+nightly-$(date 'YYYY-MM-DD' tz='Etc/UTC') >> $GITHUB_OUTPUT
```

Is this `tz=` a real syntax? It does not seem to be valid

---

Actually, running `date 'YYYY-MM-DD'` on both MacOS, debian bullseye, and ubuntu:latest fails with invalid date format.. Did this actually work correctly before?

---

OH, wow.. I didn't even notice that when writing this: the tag is using `nightly-{{date 'YYYY-MM-DD' tz='Etc/UTC'}}` vs suffix using `+nightly-$(date +'%Y%m%d')` — that is: these are not the same date commands. One is a date from GitHub Actions, the other is a date from the operating system running the actions.

---

Correction: `{{date ... }}` docker/metadata-action, which is handled by:
> date 'format' tz='Timezone' ; render date by its [moment format](https://momentjs.com/docs/#/displaying/format/). Default tz is UTC.

I'll update this PR to use 
```suggestion
          echo mastodon_version_suffix=+nightly-$(date +'%Y-%m-%d') >> $GITHUB_OUTPUT
```

But we may actually wish to calculate the date at the start of the workflow run and just use it as a constant between each step, as otherwise there's a chance that the tag name might be for one day, and the suffix for the next date, in the case of the two commands executing across the changing of days.

---

> But we may actually wish to calculate the date at the start of the workflow run and just use it as a constant between each step, as otherwise there's a chance that the tag name might be for one day, and the suffix for the next date, in the case of the two commands executing across the changing of days.

This looks like a good idea. Would you mind updating the PR doing this?